lib/galaxy/dependencies/dev-requirements.txt

```diff
@@ -48,6 +48,7 @@ commonmark==0.9.1
 contextvars==2.4; python_version < "3.7" and python_version >= "3.6"
 coverage==5.5; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version < "4"
 cryptography==3.4.7; python_version >= "3.6"
+cwltest==2.0.20200626112502; python_version >= "3.5" and python_version < "4"
 cwltool==3.0.20201109103151; python_version >= "3.6" and python_version < "4"
```

Comment:
Time to upgrade? Latest is https://pypi.org/project/cwltest/2.2.20210901154959/ & https://pypi.org/project/cwltool/3.1.20211020155521/

Reply:
as a follow on to https://github.com/galaxyproject/galaxy/pull/12858